src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java

```diff
@@ -25,6 +27,19 @@
     public final Control retries = control("maxNumRetries");
     public final Control javaPath = control("javaPath");
     public final Control jvmOptions = control("jvmOptions");
+    public final Control hostKeyVerificationStrategy = control("/");
+
+
+    //<select class="setting-input dropdownList" path="/launcher/"><option value="0">Known hosts file Verification Strategy</option><option value="1">Manually provided key Verification Strategy</option><option value="2">Manually trusted key Verification Strategy</option><option value="3">Non verifying Verification Strategy</option></select>
```

Comment:
I mainly added it as a means of copy pasting the values, but kept it around for future reference.